### PR TITLE
Fix and update of the Ukrainian translation

### PR DESCRIPTION
**Pull request by Digharatta**
Thank you for implementing the Ukrainian translation!
First live tests revealed that some strings need to be fixed, so I fixed them and also updated some other strings.

### CODE REVIEW

File: app/src/main/res/values-uk/strings.xml [hunk 1]
```diff
@@ -11,11 +11,11 @@
     <string name="account_settings_all">Усі</string>
     <string name="account_settings_avatar">Аватар</string>
     <string name="account_settings_banner">Банер</string>
-    <string name="account_settings_bio">Біографія</string>
+    <string name="account_settings_bio">Невеличкий життєпис/string>
```
Comment (MV-GH) [inline app/src/main/res/values-uk/strings.xml:14]: `/string>` is missing `<`

**Review by MV-GH — CHANGES_REQUESTED** [review]
Looks good otherwise than that small issue

File: app/src/main/res/values-uk/strings.xml (hunk 1 above)
Comment (Digharatta) [reply app/src/main/res/values-uk/strings.xml]: Thank you, fixed it.

**Review by dessalines — APPROVED** [review]

**Review by MV-GH — APPROVED** [review]

File: app/src/main/res/values-uk/strings.xml [hunk 2]
```diff
@@ -156,14 +156,14 @@
...
-    <string name="login_totp">Введіть токен двофакторної автентифікації, якщо її увімкнено</string>
-    <string name="login_password">Пароль</string>
-    <string name="login_view_model_incorrect_login">Невірний логін</string>
+    <string name="_back">Повернутися</string>
+    <string name="_email_or_username">Адреса електронної пошти або ім\'я користувача</string>
+    <string name="_instance">Примірник (інстанс)</string>
+    <string name="_instance_placeholder">наприклад: lemmy.ml</string>
+    <string name="_">Увійти</string>
+    <string name="_totp">Введіть токен двофакторної автентифікації, якщо її увімкнено</string>
+    <string name="_password">Пароль</string>
```
Comment (MV-GH) [inline app/src/main/res/values-uk/strings.xml:165]: These names are not the same

**Review by MV-GH — CHANGES_REQUESTED** [review]

**Review by MV-GH — APPROVED** [review]

**Pull request merged**